webapp/graphite/cache_metrics/views.py

```diff
@@ -0,0 +1,16 @@
+from graphite.compat import HttpResponse
+from graphite.util import json
+from graphite.carbonlink import CarbonLink
+
+
+def cache_metric(request):
+	queryParams = request.GET.copy()
```

Comment:
i was trying to combine parameters of request.GET and request.POST which needs to update.

queryParams.update(request.POST)